Tests/BluetoothTests/GATTCharacteristicTests.swift

```diff
@@ -607,52 +607,64 @@ final class GATTCharacteristicTests: XCTestCase {
         XCTAssertEqual(GATTCentralAddressResolution(data: Data([0x00])), false, "The value 0x00 should be interpreted as Not Supported")
     }
     
+
     func testBodyCompositionMeasurement() {
         
         let emptyUnit16 = UInt16(littleEndian: UInt16(bytes: (0x00, 0x00)))
         let data = Data([0x00, 0x00, 0x00, 0x00])
         
-        let SIMass = GATTBodyMass.init(rawValue: emptyUnit16, unit: MassUnit.init(rawValue: 0x2702)!)
-        let ImperialMass = GATTBodyMass.init(rawValue: emptyUnit16, unit: MassUnit.init(rawValue: 0x27B8)!)
+        typealias massUnit = GATTBodyCompositionMeasurement.MassUnit
+        typealias lengthUnit = GATTBodyCompositionMeasurement.LengthUnit
```

Comment:
@JurjDev Use uppercase on the first letter for any `typealias`

Reply:
@JurjDev Also can we remove the suffix Unit please, to make it consistent as the rest